### ISSUE TITLE
Reported issues wont fail the build

### ISSUE THREAD

**Issue by offa** [hadolint/hadolint-action#48]
Is there a way to fail the build on reported issues?

The build is reported green, even though there's eg. an error.

```yml
      - uses: hadolint/hadolint-action@v1.7.0
        with:
          dockerfile: << Path to Dockerfile here >>
          no-fail: false # Same without this line
```

I have tried setting `no-fail` to `false`, but there's no difference to without the setting.

**Comment by lorenzo** [comment]
I won't have much time to investigate this right now. If anyone want to offer help or a fix, I'm happy to merge those :)
